<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmbedDialog</class>
 <widget class="QDialog" name="EmbedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Embed</string>
  </property>
  <layout class="QVBoxLayout" name="layout_embed_dialog">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_content" native="true">
     <widget class="QComboBox" name="combo_box_guilds">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_box_channels">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_new_embed">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Kozuka Gothic Pr6N B</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New Embed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_guild">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>60</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Guild:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_channel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>60</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_edit_description">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_edit_footer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Footer</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_edit_title">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Title</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_edit_image">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>URL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_edit_thumbnail">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Thumbnail</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>EmbedDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>EmbedDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
